CMakeLists.txt

```diff
@@ -14,25 +14,4 @@ find_package(OpenVINO REQUIRED COMPONENTS Runtime)
 set(CMAKE_PROJECT_VERSION_TWEAK 0)
 set(CMAKE_PROJECT_VERSION "${OpenVINO_VERSION}.${CMAKE_PROJECT_VERSION_TWEAK}")
 
-include(GNUInstallDirs)
```

Comment:
The idea was to set common settings in one place

Reply:
Looks like we can even have a single CMakeLists.txt in the root of repo. `src/CMakeLists.txt` looks excessive